app/learning/resources.md

```diff
@@ -11,7 +11,6 @@ The below is by no means an exhaustive list of Yeoman articles, just a few that
 
 *Feel free to add additional ones, but make sure it's something of value and include the date.*
 
-* [Complete Yeoman Codelab tutorial in the Codio Web IDE](https://codio.com/docs/DEL-specifics/yeoman/) *2014-04-14*
```

Comment:
I didnt want to change the structure completely. On too long menus, when the menu aligns with the footer, by rearraging the heigth ı tried to avoid the menu's overlapping the footer. If you have any other suggestion ı can appy it